### ISSUE TITLE
Failed to upload image

### ISSUE THREAD

**Issue by bjorns** [iamjackg/md2cf#64]
I have an image notation in my markdown file 

```
![The credential set form](images/auth-token-form.png)
```

I see this error when the file is uploaded as part of a folder with the new relative links flag enabled: 

```
ERROR: Session.request() got an unexpected keyword argument 'format'    
```

I did some very brief digging and it looks like the parameters in `api.py` may not have been updated to match requests. I see on line 230:

```
                data={"comment": message} if message else None,
                format=(None, "json"),
```

and I'm thinking it should be something like:
```
                json={"comment": message} if message else None,
```

That's just a guess though.

**Comment by bjorns** [comment]
Also: Psyched for the 2.0 update, thanks!

**Comment by iamjackg** [comment]
Ugh, you're absolutely right. I need to add tests for the attachment functions. Sorry about this! I'll fix it right away.

**Comment by iamjackg** [comment]
Fixed in v2.0.1 :)
